### PR TITLE
Add @mattwthompson as a maintainer

### PR DESCRIPTION
**Pull request by mattwthompson**
Hey, that's me!

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mattwthompson** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,3 +36,4 @@ extra:
     - bgruening
     - kyleabeauchamp
     - mikemhenry
+    - mattwthompson
```
Comment (mattwthompson) [inline recipe/meta.yaml:39]: @mikemhenry you're listed here but this hasn't percolated into the repo's settings ... am I in a weird branch or something?

**Review by kyleabeauchamp — APPROVED** [review]
LGTM

**Comment by lmiq** [comment]
@bgruening 

Can this be merged? More and more people are having problems because the conda version of Packmol is outdated. It would be important to have active maintainers.

File: recipe/meta.yaml (hunk 1 above)
Comment (mikemhenry) [reply recipe/meta.yaml:39]: Wow that is weird!

**Pull request merged**